lib/db/schema/patch-026-027.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS securityEvents (
+  uid BINARY(16) NOT NULL,
+  name VARCHAR(255) NOT NULL,
```

Comment:
These names are going to come from a very small and controlled list of possible strings, and we're going to store a lot of them.  I wonder if we should use an `ENUM` column or similar here rather than `VARCHAR`.